app/src/main/java/com/checkin/app/checkin/auth/AuthViewModel.kt

```diff
@@ -17,12 +17,14 @@ class AuthViewModel(application: Application) : BaseViewModel(application) {
     private val mErrors = MutableLiveData<JsonNode>()
     private val mAuthResult = createNetworkLiveData<AuthResultModel>()
 
-    private var firebaseIdToken: String? = null
+    var firebaseIdToken: String? = null
+        private set
     private var providerIdToken: String? = null
 
     var isLoginAttempt = false
         private set
-
+    var isRegisteredAttempt = false
```

Comment:
Doesn't `isRegisteredAttempt = !isLoginAttempt` always?

Reply:
not always if login fails, still login attempt is  true

Reply:
and you need `isLoginAttempt` to be `true` while login fails because when we get the result we use this condition to check if actually a login attempt is made and it as failed
`
Resource.Status.ERROR_INVALID_REQUEST -> if (authViewModel.isLoginAttempt) {
                        if (!::phoneNo.isInitialized) {
                            toast("Invalid Mobile Number")
                            pass
                        }
                        askUserDetails()
                    }
`

Reply:
'isRegisteredAttempt = !isLoginAttempt' but i can remove isRegistedAttempt too and i will do that

Reply:
Yeah I think it's possible to do without that